docs/filter_kgraph_top_n.md

```diff
@@ -0,0 +1,59 @@
+# filter kgraph (top n)
+
+_subclass of [filter_kgraph](./filter_kgraph.md)_
+
+This operation removes kgraph edges that have attribute values are below/above top/bottom n values.
+
+### examples
+
+- [input](../examples/fill_and_filter/messages/05_filtered_kgraph_stat_input.json), [output](../examples/fill_and_filter/messages/08_filtered_kgraph_top_n_output.json)
+
+### input requirements
+
+None
+
+### output guarantees
+
+- no edges in the kgraph which have attribute values are below/above the mean +/- n standard deviations.
```

Comment:
This will need to be changed to reflect this operation ( copy paste error)

Reply:
marking as resolved since this has been fixed